translator/tocwconfig/sampleConfig/complete_linux_config.json

```diff
@@ -27,6 +26,7 @@
               "header_test": "test"
             }
           },
+          "jvm": [],
```

Comment:
Have on of these have "tomcat": ["tomcat.memory.used"] and the other "jvm": [] and maybe if you feel up to it a 3rd with no filters.